06-developer-software-guide.Rmd

```diff
@@ -39,11 +39,82 @@ files, outputs, and build and debug environments. The most popular IDEs
  remotes::install_github("ManuelHentschel/vscDebugger")
  ```
 
+ To improve the plot viewer when creating plots in `R`, install the `httpgd` package:
+
+ ```{r eval=FALSE}
+ install.packages("httpgd")
+ ``` 
+
+A number of optional settings that could be added to the user settings (settings.json) file in vscode to improve the usability of R in VS code.
+
+For example, the [settings for interacting with R terminals](https://github.com/REditorSupport/vscode-R/wiki/Interacting-with-R-terminals#sending-code-to-r-terminals) can be adjusted.
+
+Here are some that you may want to use with FIMS:
+
+```json
+{
+    // Associate .RMD files with markdown:
+    "files.associations": {
+        "*.Rmd": "markdown",
+    },
+    // A cmake setting
+    "cmake.configureOnOpen": true, 
+    // Set where the rulers are, needed for Rewrap
+    "editor.rulers": [
+        80
```

Comment:
According to the recommended limit line length from [section 7.3](https://noaa-fims.github.io/collaborative_workflow/contributor-guidelines.html#coding-good-practices), do we want to use 72 instead of 80 here?

Reply:
oh, absolutely, good point! Time to modify my own settings.json :)